src/Format/Xml.php

```diff
@@ -71,7 +71,7 @@ public function stringToObject($data, array $options = [])
     /**
      * Method to get a PHP native value for a SimpleXMLElement object. -- called recursively
      *
-     * @param  object  $node  SimpleXMLElement object for which to get the native value.
+     * @param  \SimpleXMLElement  $node  SimpleXMLElement object for which to get the native value.
```

Comment:
b/c break needs documentation and deprecation

Reply:
This is only changing the docblock to reflect the reality. There is no factual change with this.

Reply:
Has been documented.